src/app/papers.ts

```diff
@@ -93,12 +126,30 @@ export const PAPERS: Paper[] = [
       \}\n'
   },
   {
-    link: '',
+    link: 'https://link.springer.com/chapter/10.1007/978-981-19-9601-6_9',
     title: 'The Top-Down Solver: An Exercise in A²I',
```

Comment:
Apparently UT doesn't have access to this journal and I think I don't have access to this on versioncontrolseidl either. Would be nice to have a read though.

Reply:
I sent you a PDF :smile: Also, Sarah wanted to prepare an arxiv version some time later anyway, as there are some layouting things that are a bit odd.